assets/src/dashboard/app/reducer/test/stories.js

```diff
@@ -474,4 +475,77 @@ describe('storyReducer', () => {
       },
     });
   });
+
+  it(`should update isLoading when ${ACTION_TYPES.CREATING_STORY_PREVIEW_FROM_TEMPLATE} is called`, () => {
+    const result = storyReducer(
+      { ...initialState },
+      {
+        type: ACTION_TYPES.CREATING_STORY_PREVIEW_FROM_TEMPLATE,
+        payload: true,
+      }
+    );
+
+    expect(result).toMatchObject({
+      ...initialState,
+      isLoading: true,
+    });
+  });
```

Comment:
It does notify the reducer when it's not loading. It's just a rename of `LOADING_STORIES` which I believe has a true and false test for setting state. For the purposes of this state change, `isLoading` is updated to false in tandem with the response - either the previewMarkup or the error to avoid unnecessary re-renders were the data to return separate.